### DISCUSSION TITLE
problem with test and train loss curve

### DISCUSSION THREAD

**Discussion by darkacer211** [mrdbourke/pytorch-deep-learning#1334] [General]
<img width="821" height="772" alt="image" src="https://github.com/user-attachments/assets/3f56f875-72dc-43a4-a19f-f674c200fa37" />
<img width="816" height="206" alt="image" src="https://github.com/user-attachments/assets/9db7f67f-f20c-4e74-8418-6f53ead9d740" />
<img width="794" height="644" alt="image" src="https://github.com/user-attachments/assets/9f6b0f78-dac9-4de0-8251-1b19a89b08d3" />

why does it apear flat ? is it because i triained moddel too many times and the losses are constant now ??
